src/main/java/dev/jbang/cli/Run.java

```diff
@@ -210,8 +212,20 @@ String generateCommandLine(Script script) throws IOException {
 			fullArgs.add(tempFile.toString());
 		}
 
-		return String.join(" ", escapeArguments(fullArgs));
+		final String args = String.join(" ", escapeArguments(fullArgs));
 
+		// This avoids long classpath problem on Windows.
+		// @file is only available from java 9 onwards.
+		if (Util.isWindows() && JavaUtil.determineJavaVersion() >= 9) {
+			final String jarFile = script.getJar().toString();
+			final String argsFile = jarFile.substring(0, jarFile.length() - 3) + "args";
```

Comment:
As discussed on Gitter using a fixed file name is not really correct as different concurrent executions might be happening at the same time. So it should be a proper temp file.

Reply:
Done.